packages/web-api/src/controllers/health-check-controller.ts

```diff
@@ -26,20 +26,26 @@ export class HealthCheckController extends ApiController {
 
         const appInsightsClient = await this.appInsightsClientProvider();
 
+        const currentVersion = process.env.RELEASE_VERSION;
+
         const healthReport: HealthReport = {
-            buildVersion: '0.0.0',
+            buildVersion: currentVersion,
             testRuns: [],
             testsPassed: 0,
             testsFailed: 0,
         };
 
-        const e2eTestConfig = await this.serviceConfig.getConfigValue('e2eTestConfig');
-        const queryString = 'customEvents | limit 5';
-        const queryResponse = await appInsightsClient.executeQuery(queryString, e2eTestConfig.testRunQueryTimespan);
-        if (queryResponse.statusCode === 200) {
-            this.logger.logInfo(`App insights api queried with result ${JSON.stringify(queryResponse)}`);
+        if (currentVersion === undefined) {
+            healthReport.error = 'No release version number is configured';
```

Comment:
we probably should follow same approach we have with rest api and return HTTP 400 Bad Request with message body as we have in other apis

Reply:
Do we want to do that if it's a configuration issue with the function app, not a problem with the request?

Reply:
if we have problem with configuration we should send HTTP 500 instead